.travis.yml

```diff
@@ -3,10 +3,14 @@ python:
   - '2.7'
 services:
   - elasticsearch
+  - redis-server
 before_script:
   - sleep 10
 env:
-  - DATABASE_URL=postgres://postgres@localhost:5432/postgres
+  global:
```

Comment:
[Travis docs](http://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-.travis.yml)

Originally I did this:

```yaml
env:
  - DATABASE_URL=postgres://postgres@localhost:5432/postgres
  - BROKER_URL=redis://localhost:6379/4
  - HAYSTACK_URL=127.0.0.1:9200
```

However, this doesn't work because Travis won't use all three at once. It does three runs using one each. So the first run through, there's no config for Redis or Elasticsearch.

This also works:

```yaml
env:
  - DATABASE_URL=postgres://postgres@localhost:5432/postgres BROKER_URL=redis://localhost:6379/4 HAYSTACK_URL=127.0.0.1:9200
```

Both have the same effect, but I think the one-liner is uglier.